src/screens/ImportWalletScreen.tsx

```diff
@@ -34,6 +34,7 @@ interface Props {
   route: RouteProp<MainCardStackNavigatorParams, Route.ImportWallet>;
   importWallet: (wallet: Wallet, meta?: ActionMeta) => ImportWalletAction;
   wallets: Wallet[];
+  walletsLabels: string[];
```

Comment:
the `Wallet` interface already has `label:string` property. Isn't it enough?

Reply:
I wish :) - as name point is collection wallets labels from the selector